src/control/server/storage/scm/ipmctl.go

```diff
@@ -78,11 +78,11 @@ func run(cmd string) (string, error) {
 // Manage AppDirect/Interleaved memory allocation goals across all DCPMMs on a system.
 const (
 	cmdShowIpmctlVersion = "ipmctl version"
-	cmdShowRegions       = "ipmctl show -d PersistentMemoryType,FreeCapacity -region"
-	cmdCreateRegions     = "ipmctl create -f -goal PersistentMemoryType=AppDirect"
-	cmdRemoveRegions     = "ipmctl create -f -goal MemoryMode=100"
-	cmdDeleteGoal        = "ipmctl delete -goal"
-	outScmNoRegions      = "no Regions defined"
+	// command for showing regions only used for debug output
+	cmdShowRegions   = "ipmctl show -d PersistentMemoryType,FreeCapacity -region"
```

Comment:
I'm using it to get debug when the region calls don't give expected results, as explained above the tool uses a different private API so it's useful to get comparative human-readable output in unexpected conditions.

Reply:
Hmm. Okay. Honestly maybe we shouldn't switch to the API if there's something wrong with it.

Reply:
There is a delay in the response, I'm not sure there is anything wrong with it. I will get some numbers on time differences compared to master as Mike as requested on the ticket, I don't think it will be significant in the grand scheme of things given its a very infrequent activity.

Reply:
I am actively following-up with ipmctl developers.

Reply:
Thanks Tom, much appreciated.